currencyii/src/main/java/nl/tudelft/trustchain/currencyii/coin/WalletManager.kt

```diff
@@ -75,6 +81,17 @@ class WalletManager(
             }
         }
 
+        if (params == RegTestParams.get()) {
+            try {
+                val localHost = InetAddress.getByName("131.180.27.224")
+                kit.setPeerNodes(PeerAddress(params, localHost, params.port))
+                // RegTest only requires 1 peer, since the server is the only peer
+                MIN_BLOCKCHAIN_PEERS = 1
```

Comment:
Try to put as much of the information in constants above. so const val MIN_BLOCKCHAIN_PEERS_TEST, MIN_BLOCKCHAIN_PEERS_REGTEST and REG_TEST_FAUCET_IP